### DISCUSSION TITLE
can monaco content apply delta

### DISCUSSION THREAD

**Discussion by ruiminhu** [microsoft/monaco-editor#3981] [Q&A] [answered]
Hey, we are using ace editor and plan to migrate to monaco editor. 
To sync more editor instance, in ace, we use following code:
editor.on('change', e => changes.push(e));

//changes will be send by websocket, then:
changes.forEach(c => editor.getSession().getDocument().applyDelta(c));

I was wondering if there's such thing like .applyDelta function in ace, that we actually have in monaco.

Or is there a work around

**Comment by hediet** [comment, marked as answer]
There is some applyEdits (or different name) function on ITextModel!
